### PR TITLE
Add image descriptions for the relay and block producer network images

### PR DESCRIPTION
**Pull request by sempruijs**
This PR gives 2 images functional descriptions for visualy impaired coders.

### CODE REVIEW

**Review by katomm — APPROVED** [review]

**Comment by katomm** [comment]
Great idea

**Review by rphair — APPROVED** [review]
thanks @sempruijs - we have to keep remembering about accessibility ⭐

File: docs/operate-a-stake-pool/stake-pool-networking.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ Before we start with the stake pool installation and configuration, it is essent
 
 In the example below, one block producer is connected to three relays and they are connected to other relays of the Cardano network.
 
-![img](../../static/img/stake-pool-guide/stake-pool-network.jpg)
+![Block producer node is connected with 3 relayers. Those relayers are connected with the cardano network.](../../static/img/stake-pool-guide/stake-pool-network.jpg)
```
Comment (rphair) [inline docs/operate-a-stake-pool/stake-pool-networking.md:12]:
```suggestion
![Block producer node is connected with 3 relays. Those relays are connected with the cardano network.](../../static/img/stake-pool-guide/stake-pool-network.jpg)
```

File: docs/operate-a-stake-pool/stake-pool-networking.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ Before we start with the stake pool installation and configuration, it is essent
...
 
 For the operation of a stake pool, some additional nodes in the topology might be useful. For example, a back-up node for block producer is helpful in case the main block producer has an issue e.g. during an upgrade. Depending on the number of stake pools that are being managed, a monitoring node can be used to monitor them and generate alarms. Stake pool operators also need a [air-gapped system](/docs/get-started/air-gap) to store the pool cold keys. Some pool operators use a hardware wallet for storing the pool pledge.
 
 A logical topology of all the components, without considering firewalls and other security aspects can be shown as follows.
 
-![img](../../static/img/stake-pool-guide/stake-pool-setup.jpg)
+![3 relayers are connected with 1 block producer, one backup block producer and a monitoring node. The monitoring node is connected with the home pc. Their is a airgap pc, but it is not connected with other computers.](../../static/img/stake-pool-guide/stake-pool-setup.jpg)
```
Comment (rphair) [inline docs/operate-a-stake-pool/stake-pool-networking.md:18]:
```suggestion
![3 relays are connected with 1 block producer, one backup block producer and a monitoring node. The monitoring node is connected with the home PC. There is a air-gapped pc, but it is not connected with other computers.](../../static/img/stake-pool-guide/stake-pool-setup.jpg)
```
https://en.wikipedia.org/wiki/Air_gap_(networking)
https://en.wikipedia.org/wiki/Personal_computer = PC (capitalised)

**Review by rphair — COMMENTED** [review]
Just a few spelling fixes.  The terminology `relayers` is literally correct but in Cardano parlance in English we consistently call them `relays`.  Also some other spelling fixes... yes it does matter, even capitalisation, for screen readers (SEO & the "semantic web") 😎

**Review by fill-the-fill — APPROVED** [review]

**Pull request merged**